keps/sig-node/3721-support-for-env-files/README.md

```diff
@@ -365,16 +384,13 @@ No
 - Add unit tests for API validation to ensure user input fields comply with our specifications: `k8s.io/kubernetes/pkg/apis/core/validation`: `2025-06-06` - `84.7`
 - Add unit tests to validate the generation of environment variables for the pod: `k8s.io/kubernetes/pkg/kubelet`: `2025-06-06` - `70.1`
 
-##### Integration tests
```

Comment:
> We don't need integration tests, so I removed it :)

Ideally, you should explain why integration tests are not necessary rather than removing the section. I'm not going to block on this, b/c I see there's sufficient coverage in units and e2e, but please update this document in a followup PR explaining why no integration tests were written.